minio/api.py

```diff
@@ -2775,7 +2779,7 @@ def _list_objects(  # pylint: disable=too-many-arguments,too-many-branches
             response = self._execute("GET", bucket_name, query_params=query)
 
             objects, is_truncated, start_after, version_id_marker = (
-                parse_list_objects(response, bucket_name)
+                parse_list_objects(response)
```

Comment:
This comment is not related to this issue, but since you touched it:
`bucket_name` parameter of `parse_list_object` function is frivolous, and it can be removed from the function definition.

Reply:
It is there for backward compatibility.